examples/get-started/pure-js-google-maps/app.js

```diff
@@ -0,0 +1,49 @@
+/* global document, google */
+import {GoogleMapsOverlay as DeckOverlay} from '@deck.gl/google-maps';
+import {GeoJsonLayer} from '@deck.gl/layers';
+
+// source: Natural Earth http://www.naturalearthdata.com/ via geojson.xyz
+const GEOJSON =
+  'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_110m_admin_1_states_provinces_shp.geojson'; //eslint-disable-line
+
+// Retrieving GOOGLE_MAPS_API_KEY from the environment variable
+const gmUrl = `https://maps.googleapis.com/maps/api/js?key=${
```

Comment:
Nits: 
* Use capital case for constant, type out `GOOGLE_MAPS_URL`. 
* Use separate constant for the key to make it even clearer what is going on. This is the first integration hurdle for the users, after all.

```
const GOOGLE_MAPS_KEY = process.env.GOOGLE_MAPS_API_KEY;
const GOOGLE_MAPS_URL = `https://maps.googleapis.com/maps/api/js?key=${GOOGLE_MAPS_KEY}...
```

Total Nit: We are using MixedCase for mapbox environment variable in all examples, any reason to use capital case for this key?